FrameworkSystem/DB/ProxyDB.py

```diff
@@ -74,6 +84,16 @@ def __initializeDB(self):
                                      'PrimaryKey': 'Id'
                                      }
 
+    if 'ProxyDB_CleanProxies' not in tablesInDB:
+      tablesD['ProxyDB_CleanProxies'] = {'Fields': {'UserName': 'VARCHAR(64) NOT NULL',
+                                                    'UserDN': 'VARCHAR(255) NOT NULL',
+                                                    'ProxyProvider': 'VARCHAR(64) NOT NULL',
```

Comment:
Don't you need also a user group?

Reply:
No, in case when the proxy is taken from this table the group is added later